### DISCUSSION TITLE
problema smart life

### DISCUSSION THREAD

**Discussion by Franco2255** [indomus/forum#845] [HUB: Home Assistant]
Buongiorno,
 dopo ultimo aggiornamento di supervisor, alla versione core-2021-10, i componenti smart life non sono più in linea.
Cosa è cambiato?
Grazie

**Comment by Newreader** [comment]
Occorre aggiornare l'integrazione Tuya così come riportato nell'articolo di riferimento di InDomus relativo all'uscita dell'aggiornamento di HA 2021.10.0.
É necessario seguire una specifica procedura che sarà mantenuta da Tuya stessa

**Reply by Franco2255** [reply]
Grazie,
 ho risolto per le luci, anche se alcuni interruttori non mi da la dicitura esatta, mentre le tapparelle non le vede

**Comment by essegi65** [comment]
Purtroppo è stata inserita una nuova integrazione TUYA che prevede una nuova configurazione (con tra l'altro delle limitazioni in quanto mi risulta che le "cover" al momento non sono più gestite). vedi l'articolo su indomus dove si parla del nuovo aggiornamento e trovi la spiegazione. io al momento difatti ho atteso per l'aggiornamento anche io ho entità smartlife

**Comment by directy01** [comment]
L'integrazione 'Local Tuya' non è impattata vero? non sono riuscito a capirlo. Ho un solo prodotto che non funzionava con l'integrazione Tuya ma con Local Tuya si... se aggiorno non vorrei trovarmi nei pasticci.

**Reply by Franco2255** [reply]
io ne ho diversi ma non dialoga con l'app non si aggiorna se accendo da interruttore da app

**Reply by essegi65** [reply]
aggiornamento odierno. anche senza aggiornare HA le cover non sono più gestite (dentro l'applicazione smartlife invece sono perfettamente funzionanti), mentre tutti gli switch continuano a funzionare regolarmente.
Io uso l'integrazione originale, ovviamente non aggiornata, sapete se gli stessi problemi ci sono su local tuya (da hacs)?

**Reply by Franco2255** [reply]
niente da fare da Ha non aggiorna gli switch devo andare su integrazioni tuya cliccare ricarica e riavviare HA cosi si allinea

**Reply by essegi65** [reply]
Ho cancellato prima la vecchia versione dell'integrazione TUYA e poi fatto aggiornamento alla versione di HA 2021.10. seguendo quanto descritto nel sito HOMEASSISTANT (creare il progetto su TUYA iot, ecc.) e poi rimettendo l'integrazione tuya sono riuscito a avere tutti i dispositivi che comandavo prima. Ovviamente senza le cover che attualmente non sono gestite (il vecchio interruttore che le gestiva l'ho sostituito con uno shelly 2.5)

**Reply by Franco2255** [reply]
anch'io ho fatto la tua stessa procedura ma ho delle luci che si accendono in automatico ma se vado sugli switch tuya di HA non sono attivi, in più se accendo una luce da HA si accende, ma poi il pulsantino ritorna come se fosse spenta e per spegnere devo fare 2 o3 volte lo spostamento col mouse
